src/cmd/create.rs

```diff
@@ -31,9 +31,17 @@ pub struct Basic {
     force: bool,
 
     #[structopt(long)]
-    /// Use space separated seed words to create the wallet
+    /// Deprecated: Use --seed_moble.
```

Comment:
nit: s/_/-/

Reply:
nice catch. Fixed the underscore and the misspelling of `mobile`